src/app/wizard/step/provider-settings/provider/extended/component.ts

```diff
@@ -62,5 +73,22 @@ export class ProviderExtendedComponent extends BaseFormValidator implements OnIn
         this.form.removeControl(Controls.ProviderExtended);
         this.form.addControl(Controls.ProviderExtended, this._builder.control(''));
       });
+
+    this.form
+      .get(Controls.NodePortsAllowedIPRange)
+      .valueChanges.pipe(takeUntil(this._unsubscribe))
+      .subscribe(this._onNodePortsAllowedIPRangeChange.bind(this));
+  }
+
+  private _onNodePortsAllowedIPRangeChange(value: string): void {
+    if (this.Providers.AWS) {
```

Comment:
Did you actually test it for other providers and checked console logs for errors?

Reply:
I just modified this and was looking for thoughts on https://github.com/kubermatic/dashboard/pull/4198#issuecomment-1046653060

It's still WIP

Reply:
If it is work in progress mark it as draft or WIP. It was mentioned on Slack too.